test/basic.test.ts

```diff
@@ -1023,7 +1023,7 @@ describe('head tags', () => {
     // should render charset by default
     expect(indexHtml).toContain('<meta charset="utf-8">')
     // should render <Head> components
-    expect(indexHtml).toContain('<title>Basic fixture - Fixture</title>')
+    expect(indexHtml).toContain('<title>Basic fixture</title>')
```

Comment:
this seems like a regression

Reply:
It was added in https://github.com/nuxt/nuxt/pull/31169 and I believe it was a regression.

On production my website it works properly:
![image](https://github.com/user-attachments/assets/e1eaaacf-089e-4e71-8866-d81fd69ec6e6)

Reply:
the title template for the index page is set here:

https://github.com/nuxt/nuxt/blob/f209158352b09d1986aa320e29ff36353b91c358/test/fixtures/basic/plugins/my-plugin.ts#L2-L4

Shouldn't it be respected?

Reply:
Interesting... So it was a bug before updating to unhead v2? `head.vue` has specified:
```ts
      titleTemplate: title => `${title} - Title Template Fn Change`,
```
So plugin should override it?

Reply:
the title is an input to -> title template, so I don't see this conflicting.

Reply:
@danielroe It's overridden here:

https://github.com/nuxt/nuxt/blob/f209158352b09d1986aa320e29ff36353b91c358/test/fixtures/basic/pages/index.vue#L118-L121

Reply:
😱  you are quite right.

thank you for the spot...